guest/handle/api/handle.go

```diff
@@ -0,0 +1,52 @@
+/*
+   Copyright 2023 The Kubernetes Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package api
+
+import (
+	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
+	internalproto "sigs.k8s.io/kube-scheduler-wasm-extension/guest/internal/proto"
```

Comment:
If this is really api, you probalby wouldn't be depending on internal/proto. I think this will cause some package dependency problems. I would expect sigs.k8s.io/kube-scheduler-wasm-extension/guest/api/proto" and only the implementation to use the proto types. You can look at the previous functions in various places for examples of this.